modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -2002,41 +2000,64 @@ <h4><a id="node">Node</a></h4>
-          <td>applies when the <strong>disabled</strong> variable is true</td>
+          <th class="propertyname subheader" scope="col">User Action Pseudo-classes</th>
+          <th class="subheader" scope="col">Comments</th>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focused</th>
+          <th class="propertyname" scope="row">focused</th>
           <td>applies when the <strong>focused</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focus-visible</th>
+          <th class="propertyname" scope="row">focus-visible</th>
           <td>applies when the <strong>focusVisible</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">focus-within</th>
+          <th class="propertyname" scope="row">focus-within</th>
           <td>applies when the <strong>focusWithin</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">hover</th>
+          <th class="propertyname" scope="row">hover</th>
           <td>applies when the <strong>hover</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">pressed</th>
+          <th class="propertyname" scope="row">pressed</th>
           <td>applies when the <strong>pressed</strong> variable is true</td>
         </tr>
         <tr>
-        <th class="propertyname" scope="row">show-mnemonic</th>
-          <td>apples when the mnemonic affordance (typically an underscore)
-            should be shown.</td>
+          <th class="propertyname subheader" scope="col">Input Pseudo-classes</th>
+          <th class="subheader" scope="col">Comments</th>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">disabled</th>
+          <td>applies when the <strong>disabled</strong> variable is true</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">show-mnemonic</th>
+          <td>applies when the mnemonic affordance (typically an underscore) should be shown</td>
+        </tr>
+        <tr>
+          <th class="propertyname subheader" scope="col">Tree-Structural Pseudo-classes</th>
+          <th class="subheader" scope="col">Comments</th>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">first-child</th>
+          <td>applies when the node is the first child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">last-child</th>
+          <td>applies when the node is the last child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">only-child</th>
+          <td>applies when the node is the only child in its <code>Parent</code> container</td>
+        </tr>
+        <tr>
+          <th class="propertyname" scope="row">nth-child()</th>
```

Comment:
There is no "nth-child()" name, there are two: "nth-child(odd)" and "nth-child(even)".

I would recommend either to create two entries, or list two names separately.

Reply:
I've opted for `nth-child(even | odd)`.